lib/medium-admin.php

```diff
@@ -767,7 +767,7 @@ public static function cross_post($post, $medium_post, $medium_user) {
       "canonicalUrl" => $permalink,
       "license" => $medium_post->license,
       "publishStatus" => $medium_post->status,
-      "publishedAt" => mysql2date('c', $post->post_date_gmt),
+      "publishedAt" => mysql2date('c', isset($post->post_date_gmt) ? $post->post_date_gmt : $post->post_date),
```

Comment:
little known php gotcha: isset() will return true if $post->post_date_gmt is the empty string `""`. I don't know if that will ever be the case, but a safer test is:

```
$post->post_date_gmt ? $post->post_date_gmt : $post->post_date
```

which will capture anything false-y. If we were guaranteed PHP 5.3, we could even do

```
$post->post_date_gmt ?: $post->post_date
```

i.e. remove the middle portion, but WordPress requirements allows PHP 5.2.4, which doesn't have that operator extension.